src/vstart.sh

```diff
@@ -992,7 +992,7 @@ EOF
         ceph_adm config-key set mgr/cephadm/ssh_identity_pub -i ~/.ssh/id_rsa.pub
         ceph_adm mgr module enable cephadm
         ceph_adm orch set backend cephadm
-        ceph_adm orch host add $HOSTNAME
+        ceph_adm orch host add $(hostname --fqdn)
```

Comment:
```suggestion
        ceph_adm orch host add $(hostname) $(hostname -i)
```

FWIW, cephadm is using the value that is returned from python `socket.gethostname()` ...

Maybe the shell `$HOSTNAME` var is somehow set to a bare name, whereas the actual system hostname is the FQDN? e.g. `hostname == hostname --fqdn` but `$HOSTNAME != hostname` ?

Reply:
````
pdonnell@senta03 ~/ceph/build$ cat /etc/hostname
senta03
pdonnell@senta03 ~/ceph/build$ echo $HOSTNAME
senta03.front.sepia.ceph.com
pdonnell@senta03 ~/ceph/build$ hostname
senta03.front.sepia.ceph.com
pdonnell@senta03 ~/ceph/build$ hostname --fqdn
senta03.front.sepia.ceph.com
pdonnell@senta03 ~/ceph/build$ hostname -s
senta03
````

cc @djgalloway

Reply:
actually it appears the vstart script attempts to deduce the hostname and then sets the `HOSTNAME` var:

https://github.com/ceph/ceph/blob/cedf2bbd13daba6059c98046969f646b944042ae/src/vstart.sh#L1215-L1232

Reply:
but wrongly. 

```suggestion
        ceph_adm orch host add $(hostname)
```

?

Reply:
Got it, fixed.